evaluation.py

```diff
@@ -12,47 +12,66 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import go
+"""Evulation plays games between two opponents."""
```

Comment:
typo

Reply:
also that's not quite what it does -- plays games between two separate nets, reloading the weights between iterations

Reply:
Done